README.md

```diff
@@ -0,0 +1,11 @@
+# Training HuggingFace 🤗 Models on an IPU using Paperspace Gradient
+
+Whether you are looking to generate images with Stable Diffusion, derive insights from text, or need to recognize audio samples, the examples in here have you covered. And by using Paperspace to access IPUs you can be up and running in minutes!
+
+See the [README notebook](README_first.ipynb) for more detail on the tasks which are covered.
```

Comment:
This link distracts from the run on gradient button below, why have both?

Reply:
```suggestion
```

Reply:
Good point